tests/unit/e_db_mysqlTest.php

```diff
@@ -357,10 +358,29 @@ public function testDb_Close()
 */
 		public function testDelete()
 		{
-			$expected = $this->db->count('tmp');
+			// make sure the table is empty
 			$actual = $this->db->delete('tmp');
-			$this->assertEquals($expected, $actual, 'Unable to delete all records.');
+			$this->assertNotEmpty($actual, 'Unable to empty the table.');
```

Comment:
There should be a second call, perhaps to `e_db_mysql::rowCount()` to make sure that the table is empty instead of trusting the word of `e_db_mysql::delete()`.